dev-tools/packer/xgo-scripts/before_build.sh

```diff
@@ -33,15 +33,15 @@ cp $BEATNAME.template-es2x.json $PREFIX/$BEATNAME.template-es2x.json
 
 # linux
 cp $BEATNAME.yml $PREFIX/$BEATNAME-linux.yml
-
-# binary
-cp $BEATNAME.yml $PREFIX/$BEATNAME-binary.yml
```

Comment:
Ok, makes sense now. We don't need the binary anymore as we also use linux for the binary one. Ignore my 2 comments.